orbit/pkg/update/notifications.go

```diff
@@ -501,14 +501,14 @@ func (w *windowsMDMBitlockerConfigFetcher) attemptBitlockerEncryption(notifs fle
 		ClientError:   bitlockerError,
 	}
 
+	errServerUpdate := w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
-		log.Error().Err(err).Msg("failed to encrypt the volume")
+		log.Error().Err(errServerUpdate).Msg("failed to send encryption result to Fleet Server")
 		return
 	}
 
-	err = w.EncryptionResult.SetOrUpdateDiskEncryptionKey(payload)
 	if err != nil {
```

Comment:
So this condition here on line 510 is to handle the error found at line 490 above IIUC, right? I think it would be helpful to add a comment mentioning that, as at a glance it looks like an extra, forgotten/unused error check.

Reply:
Thanks, I've just added comments to better explain this